Lesson4.ipynb

```diff
@@ -29,6 +33,1310 @@
     "\n",
     "Select **Lesson4.ipynb**"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read csv file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What is csv file?\n",
+    "\n",
+    "A CSV is a comma separated values file which allows data to be saved in a table structured format. CSVs are a kind of spreadsheet same with Excel but with a .csv extension. Traditionally they take the form of a text file containing information separated by commas, hence the name.\n",
+    "\n",
+    "** Content in CSV Sample **\n",
+    "\n",
+    "The followings are the part of data from Biggest Loser:\n",
+    "\n",
+    "team_no,team_name,team_captain,team_member,2018-04-02,2018-04-03\n",
+    "\n",
+    "1,TBD,FALSE,1-1,11980,10437\n",
+    "\n",
+    "1,TBD,FALSE,1-2,22935,13399\n",
+    "\n",
+    "However, csv file is a table, as we are familiar with Excel, to make it more clear, we can have a comparison with Excel we generally use in our daily work, the first row is the coloumn name. Start from the second row, they are the specific information of each person.\n",
```

Comment:
Replace "However, csv file is a table, as we are familiar with Excel, to make it more clear, we can have a comparison with Excel we generally use in our daily work, the first row is the coloumn name. Start from the second row, they are the specific information of each person." with
"In the CSV format, the first header row consists of column names, while each subsequent row shows the data for each team member separated by a comma."